homeassistant/components/switch/pilight.py

```diff
@@ -73,6 +78,24 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     add_devices(devices)
 
 
+class _ReceiveHandle(object):
```

Comment:
Why a class that needs a switch object to work? This is mixing functional programing and object oriented programing.